R/init-package.R

```diff
@@ -16,4 +16,7 @@ initPackage <- function() {
   apiConfig$dimensionFilePath <- filePathFor("OSPSuite.Dimensions.xml")
 
   rClr::clrCallStatic("OSPSuite.R.Api", "InitializeOnce", apiConfig$ref)
+
+  # initialize global variables (mostly usef for performance optimization)
+  ospsuiteEnv$containerTask <- rClr::clrCallStatic("OSPSuite.R.Api", "GetContainerTask")

```

Comment:
`ospsuiteEnv` is our global environment for global stuff. it won't be exposed by the package.
Does this sound ok to you?

Reply:
Sounds good.